_posts/2019-07-15-whats-new-for-tizen-on-forms-410.md

```diff
@@ -10,50 +10,42 @@ toc_sticky: true
+`Xamarin.Forms 4.1.0` has been released on NuGet. This post introduces the new Tizen features on `Xamarin.Forms 4.1.0`. <br/>
+Many new UI features were added on `Xamarin.Forms 4.0`, including Shell, CollectionView and Material Visual, which are supported on the Tizen platform.
 
+## 1. Shell
+The `Xamarin.Forms` Shell feature reduces the complexity of building mobile apps by providing fundamental app architecture features.
 
-I am pleased to tell the major new UI features including Shell, CollectionView and Material Visual are supported on Tizen platform.<br/>
-`Xamarin.Forms 4.1.0` includes Tizen support for the new UI features, and I'd like to show each features running on Tizen platform.
+For more details on Shell, see [Shell navigation documentation](https://docs.microsoft.com/en-us/xamarin/xamarin-forms/app-fundamentals/shell/introduction)
 
-
-## 1.Shell
-Shell is the main feature of new Xamarin.Forms. It seeks to reduce the complexity of building mobile apps by providing fundamental app architecture features.
-
-For more details on Shell, you can check out [Shell navigation documentation](https://docs.microsoft.com/en-us/xamarin/xamarin-forms/app-fundamentals/shell/introduction)
-
-
-Shell provides 3 types of common mobile navigation UI:
+Shell provides three types of common mobile navigation:
  - Flyout
  - Bottom Tabs
- - Top Tabs <br/>
+ - Top Tabs
 
 ![][shell]
  
 ## 2. CollectionView
-CollectionView is a view for presenting lists of data using different layout specifications. It aims to provide a more flexible, and performant alternative to ListView.
+CollectionView is used to present lists of data using different layout specifications. It provides a more flexible and performant alternative to ListView.
 
-For more details on CollectionView, you can check out [CollectionView documentation](https://docs.microsoft.com/en-us/xamarin/xamarin-forms/user-interface/collectionview/)<br/>
+For more details on CollectionView, see [CollectionView documentation](https://docs.microsoft.com/en-us/xamarin/xamarin-forms/user-interface/collectionview/)<br/>
 
 ![][collectionview1]
 ![][collectionview2]
 ![][collectionview3]
 
 ## 3. Material Visual
 
-Material Design is an opinionated design system created by Google, that prescribes the size, color, spacing, and other aspects of how views and layouts should look and behave.
+Material Design is an opinionated design system created by Google, which prescribes the size, color, spacing, and other aspects of how views and layouts should look and behave.
 
-Xamarin.Forms provides the way to apply material design across the platforms to controls in Xamarin.Forms apps.
-This is achieved with material renderers, that apply the Material Design rules.
+Xamarin.Forms provides the way to apply material design across the platforms to controls in Xamarin.Forms apps. Material renderers apply the Material Design rules.
```

Comment:
For the last sentence, isn't it better to keep the original one to be more clear?
"This is achieved with material renderers, that apply the Material Design rules."

Reply:
OK. But please remove the comma - it isn't necessary.

Reply:
Ok! applied!